### DISCUSSION TITLE
Which is better? HTML or MarkDown?

### DISCUSSION THREAD

**Discussion by karaka200** [doctrine/doctrine2#9342] [Support Questions] [answered]
Hi

I have 3 questions:

1- In your opinion, which is better to store the texts of the site blog in the database? HTML or MarkDown?

2- If use HTML, does the high volume of stored characters slow down the retrieval of information from the database?

3- If use MarkDown, will converting it to HTML on the server side cause additional requests?

Thank yoy

**Comment by obw** [comment, marked as answer]
I would use Markdown for storage!

The generated HTML should be cached!

The pros of storing Markdown are:

- Uses a little less storage.
- It's more secure (When you deny inline HTML)
- It's easer to make Layout changes

HTML is faster, just get it and send it. But also, when you must think about the Rendering Time, you will always need some form of caching!
